build-logic/src/main/groovy/buildlogic.reproducible-archives.gradle

```diff
@@ -0,0 +1,5 @@
+// See https://docs.gradle.org/current/userguide/working_with_files.html#sec:reproducible_archives
+tasks.withType(AbstractArchiveTask).configureEach {
+    preserveFileTimestamps = false
```

Comment:
🤔 If you use normalization, I suppose you can remove the reproducible JARs here again, right?